src/puppetlabs/puppetdb/http/nodes.clj

```diff
@@ -27,34 +27,27 @@
       (http/status-not-found-response "node" node))))
 
 (defn routes
-  [version]
+  [{:keys [scf-read-db url-prefix api-version] :as globals}]
   (app
    []
-   (http-q/query-route :nodes version http-q/restrict-query-to-active-nodes')
+   (http-q/query-route :nodes globals http-q/restrict-query-to-active-nodes')
 
    [node]
-   {:get
-    (-> (fn [{:keys [globals]}]
-          (node-status version
-                       node
-                       (:scf-read-db globals)
-                       (:url-prefix globals)))
-        ;; Being a singular item, querying and pagination don't really make
-        ;; sense here
-        (validate-query-params {}))}
+   {:get (-> (node-status api-version node scf-read-db url-prefix)
+             constantly
```

Comment:
I actually just meant

```
(-> (constantly (node-status ...
```
although using fn or # you could probably get it all on one line. I also just discovered this, but the semantics aren't the same between (fn []) and constantly. Don't think it'll bite us here but it's a good thing to know

```
puppetlabs.puppetdb.core=> (def x 10)
#'puppetlabs.puppetdb.core/x
puppetlabs.puppetdb.core=> (def foo (constantly x))
#'puppetlabs.puppetdb.core/foo
puppetlabs.puppetdb.core=> (def bar (fn [] x))
#'puppetlabs.puppetdb.core/bar
puppetlabs.puppetdb.core=> (foo)
10
puppetlabs.puppetdb.core=> (bar)
10
puppetlabs.puppetdb.core=> (def x 11)
#'puppetlabs.puppetdb.core/x
puppetlabs.puppetdb.core=> (foo)
10
puppetlabs.puppetdb.core=> (bar)
11
```